Library/Aliases/neo4j-client.rb

```diff
@@ -0,0 +1 @@
+../Formula/libneo4j-client.rb
```

Comment:
Only because it installs a binary called neo4j-client. If you don't think it's necessary, I'll drop it!